server/cluster/cluster.go

```
@@ -1651,16 +1659,39 @@ func (c *RaftCluster) RemoveStoreLimit(storeID uint64) {
 	}
 	delete(cfg.StoreLimit, storeID)
 	c.opt.SetScheduleConfig(cfg)
+	var err error
+	for i := 0; i < persistLimitRetryTime; i++ {
+		if err = c.opt.Persist(c.storage); err == nil {
+			return
+		}
+	}
+	log.Error("persist store limit meet error", errs.ZapError(err))
 }
 
 // SetStoreLimit sets a store limit for a given type and rate.
 func (c *RaftCluster) SetStoreLimit(storeID uint64, typ storelimit.Type, ratePerMin float64) {
```

Need to return an error and handle it?